draft-ietf-ohai-ohttp.md

```diff
@@ -1357,17 +1357,29 @@ the chosen key configuration. The Oblivious Relay and Gateway resources can link
 requests using the same key configuration by matching KeyConfig.key\_id, or, if
 the Target Resource is willing to use trial decryption, a limited set of key
 configurations that share an identifier. An Oblivious Relay Resource can link
-requests using the public key corresponding to KeyConfig.key\_id.
-
-Request resources are capable of linking requests depending on how KeyConfigs
-are produced by servers and discovered by clients. Specifically, servers can
-maliciously construct key configurations to track individual clients. A specific
-method for a client to acquire key configurations is not included in this
-specification. Clients need to consider these tracking vectors when choosing a
-discovery method.  Applications using this design should provide accommodations
-to mitigate tracking using key configurations.
-{{?CONSISTENCY=I-D.wood-key-consistency}} provides an analysis of the options
-for ensuring the key configurations are consistent between different clients.
+requests using the public key corresponding to KeyConfig.key\_id or via the
+Oblivious Gateway Resource. We refer to this set of information as the gateway's
```

Comment:
```suggestion
identity of the Oblivious Gateway Resource. We refer to this set of information as the gateway's
```

Reply:
Or
```suggestion
URI of the Oblivious Gateway Resource. We refer to this set of information as the gateway's
```

Reply:
Right, I just left identity and URI out so it generally refers to "the resource." I'm OK with URI though.